Source/WebCore/page/RemoteFrameView.h

```diff
@@ -36,6 +36,7 @@ class RemoteFrameView final : public FrameView {
     static Ref<RemoteFrameView> create(RemoteFrame& frame) { return adoptRef(*new RemoteFrameView(frame)); }
 
     Type viewType() const final { return Type::Remote; }
+    RemoteFrame& frame() const { return m_frame.get(); }
```

Comment:
Can we do something like this instead?

const RemoteFrame& frame() const { return m_frame.get(); }
RemoteFrame& frame() { return m_frame.get(); }

That makes it so if you have a const RemoteFrameView&, you can't mutate its frame.